CountryApp/CountryApp/RegisterViewController.m

```diff
@@ -0,0 +1,143 @@
+//
+//  RegisterViewController.m
+//  CountryApp
+//
+//  Created by Artak Martirosyan on 5/16/18.
+//  Copyright © 2018 Artak. All rights reserved.
+//
+
+#import "RegisterViewController.h"
+#import "Helper.h"
+#import "UIViewController+ErrorHandler.h"
+#import "AccountManager.h"
+
+@interface RegisterViewController ()
+@property (weak, nonatomic) IBOutlet UIScrollView *contentScrollView;
+@property (weak, nonatomic) IBOutlet UITextField *firstNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *usernameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordTextField;
+@property (weak, nonatomic) IBOutlet UITextField *countryNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *capitalNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *languageTextField;
+
+@property (weak, nonatomic) UITextField *activeTextField;
+
+@end
+
+@implementation RegisterViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+- (IBAction)backAction:(id)sender {
+    [self.navigationController popViewControllerAnimated:true];
+}
+
+-(void) viewWillAppear:(BOOL)animated{
+    [super viewWillAppear:animated];
+  //  [self.navigationController setNavigationBarHidden:false];
+    [self registerForKeyboardNotifications];
+}
+- (IBAction)registerAction:(id)sender {
+    Account * account = [[Account alloc] init];
```

Comment:
It will be good to have a validation checker in the Helper.m class and do all this stuff in that class and in ViewController class call only something like validateFields or something like that. 
Also as this Helper class contains only a validation logic, it will be good idea to rename it to Validation or, registration validation as you may need Helper name for other purposes